src/GafferML/Tensor.cpp

```diff
@@ -213,6 +216,16 @@ Tensor::Tensor( const IECore::ConstDataPtr &data, std::vector<int64_t> shape )
 				std::copy( typedData->readable().begin(), typedData->readable().end(), value.GetTensorMutableData<bool>() );
 				m_state = new State{ std::move( value ), nullptr };
 			}
+			else if constexpr( std::is_same_v<DataType, StringVectorData> )
+			{
+				// Special case for the vector of std::string fiasco.
```

Comment:
The "vector of bool fiasco" is a specific problem with `vector<bool>` that doesn't apply to strings. It looks like we do need a special case for strings, but not because of any underlying problems with `vector<string>`.

Reply:
addressed here c7db62db2b5f16670e52e05837488387164ce711